packages/components/src/icon-button/stories/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import IconButton from '../';
+
+export default { title: 'IconButton', component: IconButton };
+
+export const _default = () => <IconButton icon="ellipsis" label="More" />;
+
+export const Grouped = () => {
+	const GroupContainer = ( props ) => (
+		<div style={ { display: 'inline-flex' } } { ...props } />
+	);
```

Comment:
We could remove the props forwarding here to make things simpler.

Reply:
Yes! Can do. Thank you! <3